src/gradualizer.erl

```diff
@@ -6,6 +6,10 @@
 %%%   in the given directory.
 %%% - `print_file': if `true' prefix error printouts with the file name the
 %%%   error is from.
+%%% - `crash_on_error`: if `true` crash on the first produced error
+%%% - `no_report_errors`: if `true` don't print type checking error messages
+%%% - `return_errors`: if `true`, errors (in their internal format) are returned
```

Comment:
what if it would not be possible to both print and return errors? (or does anyone want to do that?) just a single option `return_errors`, when false, errors are printed, as by default. (dialyzer has a similar option `--raw`)

Reply:
I agree with Péter that *report* can just be the negation of *return*. No separate option needed IMO.

If we do have both options, I think the non-negated `report_errors` is better than `no_report_errors`. The default value for *report* can be the negation of *return*, so that you don't get both unless you specify both.